creusot-contracts/src/std/ops.rs

```diff
@@ -175,19 +175,22 @@ extern_spec! {
             trait FnOnce<Args> where Args: Tuple {
                 #[requires(self.precondition(arg))]
                 #[ensures(self.postcondition_once(arg, result))]
-                fn call_once(self, arg: Args) -> Self::Output;
+                fn call_once(self, arg: Args) -> Self::Output
+                    where Self: Sized;
```

Comment:
This is required by the `requires`/`ensures`. Do we want `precondition` and `postcondition` to take `&self` instead?

Reply:
Oh but probably the blanket *impl* assumes this. What if you add `?Sized` to it?

Reply:
```
error[E0277]: the size for values of type `Self_` cannot be known at compilation time
   --> creusot-contracts/src/std/ops.rs:177:27
    |
176 |                 #[requires(self.precondition(arg))]
    |                 - this type parameter needs to be `Sized`
177 |                 #[ensures(self.postcondition_once(arg, result))]
    |                           ^^^^ doesn't have a size known at compile-time
    |
    = note: all values captured by value by a closure must have a statically known size
help: consider removing the `?Sized` bound to make the type parameter `Sized`
    |
175 -             trait FnOnce<Args> where Args: Tuple {
176 -                 #[requires(self.precondition(arg))]
175 +             trait [requires(self.precondition(arg))]
    |
```